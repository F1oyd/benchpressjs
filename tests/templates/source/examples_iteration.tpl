<!-- BEGIN arr -->
{@index} = {@value}
<!-- END -->

{{{ each arr }}}
{@index} = {@value}
{{{ end }}}



<!-- BEGIN people -->
{people.name} is {../age} years old.
<!-- END -->

{{{ each people }}}
{people.name} is {../age} years old.
{{{ end }}}



<!-- BEGIN usernames -->
#{@index}   {@key}: {@value}
<!-- END -->

{{{ each usernames }}}
#{@index}   {@key}: {@value}
{{{ end }}}



<!-- BEGIN admins -->
username: {@key}
name: {admins.name}

<!-- END -->

{{{ each admins }}}
username: {@key}
name: {admins.name}

{{{ end }}}