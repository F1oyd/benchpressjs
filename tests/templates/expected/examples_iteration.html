0 = a
1 = b
2 = c
3 = d
4 = e

0 = a
1 = b
2 = c
3 = d
4 = e



John Smith is 34 years old.
Samantha Walker is 67 years old.
Josh Hawkins is 12 years old.

John Smith is 34 years old.
Samantha Walker is 67 years old.
Josh Hawkins is 12 years old.



#0   jumpbugger: Jacob Harley
#1   neatoooo: Kate Worden
#2   hamster: Nate Francis

#0   jumpbugger: Jacob Harley
#1   neatoooo: Kate Worden
#2   hamster: Nate Francis



username: jumpbugger
name: Jacob Harley

username: neatoooo
name: Kate Worden

username: hamster
name: Nate Francis

username: jumpbugger
name: Jacob Harley

username: neatoooo
name: Kate Worden

username: hamster
name: Nate Francis